src/kimmdy/runmanager.py

```diff
@@ -772,21 +819,56 @@ def _apply_recipe(self, files: TaskFiles) -> TaskFiles:
             write_time_marker(files.outputdir / MARK_FAILED, "failed")
             raise RuntimeError(m)
 
+        if isinstance(self.kmcresult, KMCReject):
```

Comment:
when is this block executed?

Reply:
true, that needs different handling now. fixed.